src/stories/Layout.stories.js

```diff
@@ -1,48 +1,41 @@
+import React from 'react'
 import { fn } from '@storybook/test'
-import { Button } from './Button'
+import { Layout } from '../Components/Layout/layout'
+
+const LayoutExample = () => {
```

Comment:
Is this copy of https://github.com/kse-web-development-practice/example/blob/main/src/stories/Layout.stories.js ?